src/mjolnir/ingest_transit.cc

```diff
@@ -158,22 +164,25 @@ std::priority_queue<tile_transit_info_t> select_transit_tiles(const boost::prope
     return tile_map.insert({graphid, tile_transit_info_t{graphid}}).first->second;
   };
 
-  filesystem::recursive_directory_iterator gtfs_feed_itr(path);
+  filesystem::recursive_directory_iterator gtfs_feed_itr(gtfs_path);
   filesystem::recursive_directory_iterator end_file_itr;
   for (; gtfs_feed_itr != end_file_itr; ++gtfs_feed_itr) {
-    if (filesystem::is_directory(gtfs_feed_itr->path())) {
-      auto feed_path = gtfs_feed_itr->path().string();
-      LOG_INFO("Loading " + feed_path);
-      gtfs::Feed feed(feed_path);
+    const auto& feed_path = gtfs_feed_itr->path();
+    if (filesystem::is_directory(feed_path)) {
+      // feed_path has a trailing separator
+      const auto feed_name = feed_path.filename().string();
```

Comment:
this is the change where I take the feed name, not its full path, to create the tile_info objects